slamd/discovery/processing/experiment/slamd_random_forest.py

```diff
@@ -12,12 +12,12 @@ class SlamdRandomForest(RandomForestRegressor):
     minimum data requirements
     """
 
-    def fit(self, x, y, weights=None):
+    def fit(self, x, y, weights=None, random_seed=42):
         if y.shape[0] < LOLOPY_MINIMUM_DATA_POINTS:
             x = np.tile(x, (4, 1))
```

Comment:
I just assumed that this is good for now. This way it is fully reproducable just like Gauss is. However, in case that this should be changed we can still discuss.